dev-cpp/benchmark/benchmark-1.9.0.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_12 )
+
+inherit cmake-multilib python-single-r1
+
+DESCRIPTION="A microbenchmark support library"
+HOMEPAGE="https://github.com/google/benchmark/"
+SRC_URI="https://github.com/google/benchmark/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="Apache-2.0"
+SLOT="0/$(ver_cut 1)"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~loong ~ppc ~ppc64 ~riscv ~x86"
+IUSE="doc +exceptions libcxx libpfm lto test +tools"
+RESTRICT="!test? ( test )"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="
+	libcxx? ( sys-libs/libcxx[${MULTILIB_USEDEP}] )
+	libpfm? ( dev-libs/libpfm:= )
+"
+
+BDEPEND="
+	>=dev-build/cmake-3.10
+	test? ( dev-cpp/gtest[${MULTILIB_USEDEP}] )
+"
```

Comment:
You need doxygen dep with `+doc` use flag here.

Reply:
Done!